src/Essentials/samples/Samples/App.xaml.cs

```diff
@@ -27,8 +27,6 @@ public App()
 
 			VersionTracking.Track();
 
-			MainPage = new NavigationPage(new HomePage());
```

Comment:
Can we use MainPage here still? My PR was merged so we should probably use that to make sure the migration is good.

Reply:
👍